clients/search-component/src/TrieveModal/index.css

```diff
@@ -319,7 +313,7 @@ body {
       }
 
       .system {
-        @apply mt-2 sm:mt-8 mb-5 items-center overflow-ellipsis max-w-[100vw] sm:max-w-[330px];
+        @apply mt-2 sm:mt-8 mb-5 items-center overflow-ellipsis max-w-[100vw] sm:max-w-[530px];
```

Comment:
Can we make this an ecomm only change? I think there was some reasoning for it on the site/docs mode.